lib/pxutil.js

```diff
@@ -140,12 +140,31 @@ function filterConfig(config) {
     return jsonConfig;
 }
 
+/**
+ * extractCookieNames - Extract all the cookie names that were sent in the cookie http header.
+ * @param {Object} request - The received http request.
+ * @param {Object} pxCtx - current request context.
+ */
+function extractCookieNames(cookieHeader) {
+    var result;
+    if(cookieHeader){
+        var cookies = cookieHeader.split(';');
+        var cookieNames = new Array (cookies.length);
+        for (var i = 0; i < cookies.length ; i++){
```

Comment:
a single javascript line can do all this at once:

```suggestion
        const cookieNames = cookies.map(cookie => cookie.split('=')[0].trim());
```

Reply:
fixed